### PR TITLE
Added implementation for euclidean algorithm in PHP

### PR DESCRIPTION
**Pull request by PudottaPommin**

### CODE REVIEW

**Comment by PudottaPommin** [comment]
@Butt4cak3 good to go

File: contents/euclidean_algorithm/code/php/euclidean.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+function euclid_sub(float $a, float $b): float
```
Comment (Butt4cak3) [inline contents/euclidean_algorithm/code/php/euclidean.php]: The function arguments are supposed to be `int`.

File: contents/euclidean_algorithm/code/php/euclidean.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    return $a;
+}
+
+function euclid_mod(float $a, float $b): float
+{
+    $a = abs($a);
+    $b = abs($b);
+
+    while ($b != 0)
```
[the remark this answers is not in the document]
Comment (PudottaPommin) [reply contents/euclidean_algorithm/code/php/euclidean.php]: Yes, **but** [abs](http://php.net/manual/en/function.abs.php#refsect1-function.abs-returnvalues)
`The absolute value of number. If the argument number is of type float, the return type is also float, otherwise it is integer (as float usually has a bigger value range than integer).`

If I change it as you wanted, we have to use `(float) abs($a)` and `(float) abs($b)` to enforce float numbers.

**Comment by Butt4cak3** [comment]
Looks fine to me. Good job!

**Pull request merged**